Pages/CourseList/script.js

```diff
@@ -0,0 +1,76 @@
+document.addEventListener('DOMContentLoaded', () => {
+    // Define the API endpoint
+    const apiEndpoint = 'http://devpass-backend-production.up.railway.app/api/v1/cursos/cursos';
```

Comment:
```suggestion
    const apiEndpoint = `{baseUrl}/cursos/cursos`;
```
Aqui deve pegar a api usando a baseUrl que tá em config.js, caso não tenh ainda faz um merge da main pra tua e usa